Networking/Networking/Model/OrderItem.swift

```diff
@@ -113,7 +113,7 @@ public struct OrderItem: Decodable, Equatable, Hashable, GeneratedFakeable, Gene
 
 /// Defines all of the OrderItem's CodingKeys.
 ///
-private extension OrderItem {
+internal extension OrderItem {
```

Comment:
nit: since `internal` is default access level, do we need to declare it explicitly?